resolwe/flow/executors/startup_processing_container.py

```diff
@@ -221,9 +221,16 @@ def process_script(self, message):
             yield from self.manager.upload_log_files()
 
             if KEEP_DATA:
-                yield from self.manager.send_file_descriptors(
-                    [str(file_) for file_ in Path("./").rglob("*") if file_.is_file()]
-                )
+                try:
+                    yield from self.manager.send_file_descriptors(
+                        [
+                            str(file_)
```

Comment:
`file` is not a keyword in Python 3, so there is no need for the underscore.

Reply:
Good point, much nicer code (in a single line).